src/webapi/custom_elements.rs

```diff
@@ -0,0 +1,62 @@
+use webcore::value::Reference;
+webapi::error::TypeError;
+
+use webapi::dom_exception::{
+    NotSupportedError,
+    SyntaxError
+};
+
+#[cfg(feature = "experimental_features_which_may_break_on_minor_version_bumps")]
+use webcore::promise::{Promise, TypedPromise};
+
+/// The CustomElementRegistry interface provides methods for registering custom elements 
+/// and querying registered elements. To get an instance of it, use the window.customElements property. 
+///
+/// [(JavaScript docs)](hhttps://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry)
+/// https://html.spec.whatwg.org/multipage/custom-elements.html#customelementregistry
```

Comment:
Links to the specs should be in a normal comment, that is `//` instead of `///`. (: